lib/origen/parameters/set.rb

```diff
@@ -61,11 +61,12 @@ def method_missing(method, *args, &block)
             self[m] = args.first
           end
         else
+          # binding.pry if $stop == 1
           if args.length != 0
             super
           else
             if !key?(method)
-              super
+              nil
```

Comment:
@info-rchitect not sure I understand the need for this?

Reply:
Without that change this spec test will fail:

~~~ruby
ip.params(:not_chain).chain.should == nil
~~~

Reply:
if I understand correctly, it'll also effect the min and max HASH methods?

ie ip.params(:it_is_a_range).max will return a nil? I could be wrong but i just want to understand

Reply:
let me check that test case

Reply:
I updated the code and the spec tests.

Reply:
@info-rchitect thanks, it covers all the possible uses cases we have internally, looks good to me.

@ginty thoughts?